django-stubs/views/decorators/__init__.pyi

```diff
@@ -0,0 +1,13 @@
+from typing import Any, Protocol, TypeVar
+
+from django.http.request import HttpRequest
+from django.http.response import HttpResponseBase
+
+# `*args: Any, **kwargs: Any` means any extra argument(s) can be provided, or none.
+class _View(Protocol):
+    def __call__(self, request: HttpRequest, /, *args: Any, **kwargs: Any) -> HttpResponseBase: ...
```

Comment:
Looks like defining a Protocol for `request` wouldn't work either.

I'm open to just hinting as `request: Any` to solve the `AuthenticatedHttpRequest` hack. That's basically the approach we had until now.